frontend/dispatcher.lua

```diff
@@ -1063,6 +1091,10 @@ function Dispatcher:_showAsMenu(settings, exec_props)
             callback = function()
                 UIManager:close(quickmenu)
                 Dispatcher:execute({[v.key] = settings[v.key]})
+                if keep_open_on_apply then
+                    quickmenu:setTitle(title)
+                    UIManager:show(quickmenu)
+                end
```

Comment:
I imagine that's so as not to break #11122 too badly? ;p.

I would perhaps add a comment somewhere to prevent someone from thinking "well, why don't we just skip the first close instead?" down the line ;).

Reply:
(1) If touch input is off when a QM-with-keep-open is called, touch input goes on and remains on after applying any action from this QM.

(2) If QM-with-keep-open has an action "Disable touch input" and we press it, QM remains on the screen but the touch input is off, so we are locked (sleep/wakeup required).
I believe (2) is an edge case and can be left as is.